test/sortBy.test.js

```diff
@@ -0,0 +1,15 @@
+const { sortBy } = require("../lib");
+
+describe("sortBy", () => {
+  test("ascending order from [3, 1, 2] should be [1, 2, 3]", () => {
+    expect(sortBy((a, b) => a > b, [3, 1, 2])).toBe([1, 2, 3]);
```

Comment:
likely need to use `toEqual` here otherwise it checks for the same reference variable and not the equality of the values.